api/lib/getJSON.js

```diff
@@ -1,4 +1,4 @@
-const object = require('lodash/fp/object');
+const getObject = require('lodash/get');
```

Comment:
just fyi @ManjotS looks like `fp/object` lib has been deprecated -> its not available in the main master branch of lodash anymore

Reply:
never mind i spoke too soon, it exists under this branch `https://github.com/lodash/lodash/tree/4.17.15-post/fp` which is the same version we're using

Reply:
point still stands that the `object.js` file no long exists